src/main/java/com/cdancy/bitbucket/rest/features/PullRequestApi.java

```diff
@@ -157,4 +162,50 @@ CommitPage commits(@PathParam("project") String project,
                                 @Nullable @QueryParam("withCounts") Boolean withCounts,
                                 @Nullable @QueryParam("limit") Integer limit,
                                 @Nullable @QueryParam("start") Integer start);
+
+    @Named("pull-request:list-activities")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45888278197104"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/activities")
+    @Fallback(PullRequestOnError.class)
+    @GET
+    ActivitiesPage listActivities(@PathParam("project") String project,
+                              @PathParam("repo") String repo,
+                              @PathParam("pullRequestId") long pullRequestId,
+                              @Nullable @QueryParam("limit") Integer limit,
+                              @Nullable @QueryParam("start") Integer start);
+
+    @Named("pull-request:list-participants")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45627978405632"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/participants")
+    @Fallback(PullRequestOnError.class)
```

Comment:
Lets create an `ParticipantsPageOnError` class to handle this and ensure we have a test to stress it.